src/data/types.ts

```diff
@@ -56,6 +56,16 @@ export type Project = {
      */
     website?: string;
 
+     /**
+     * Link to deployed app on a store
+     */
+    appDownloadLink?: string;
+
+    /**
+     * Type of app
+     */
+    appType?: AppType;
```

Comment:
for anything with an option, let's nest it to avoid proliferating options and make it clear what options are associated, and standardize on `url` instead of `link`:

```
app?: {
    url: string;
    type: 'ios' | 'android';
};
```

Reply:
done!